wpilibNewCommands/src/main/native/include/frc2/command/CommandHelper.h

```diff
@@ -34,7 +34,9 @@ class CommandHelper : public Base {
   }
 
  protected:
-  std::unique_ptr<Command> TransferOwnership() && override {
+  [[deprecated("Use ToPtr() instead")]]
+      std::unique_ptr<Command> TransferOwnership() &&
+      override {
```

Comment:
Wow that formatting is ugly, use WPI_DEPRECATED() here too.  I think it's the `&&` messing it up in this case?